src/js/header/zoom.mjs

```diff
@@ -0,0 +1,9 @@
+export default function showZoomMenu() {
+  const zoomBtn = document.querySelector("#zoomBtn");
+  const zoomMenu = document.querySelector("#zoomMenu");
+  if (zoomBtn) {
+    zoomBtn.addEventListener("click", () => {
+      zoomMenu.classList.toggle("hidden");
+    })
+  }
+}
```

Comment:
```suggestion
export default function showFileMenu() {
  const fileBtn = document.querySelector('#fileBtn');
  const fileMenu = document.querySelector('#fileMenu');
  if (fileBtn) {
    fileBtn.addEventListener('click', (e) => {
    ; /* e.stopPropagation() is called to prevent the closeClickOutside from being triggered on initial click  */
      e.stopPropagation()
      fileMenu.classList.toggle('hidden');
      closeClickOutside(fileMenu);
    });
  }
}

 function closeClickOutside(ele) {
  const clickOutside = document.addEventListener('click', (e) => {
    if (ele.contains(e.target)) return;
    ele.classList.add('hidden');
    document.removeEventListener('click', clickOutside);
  });
}

```